nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/package.json

```diff
@@ -7,13 +7,16 @@
     "test": "test"
   },
   "devDependencies": {
-    "@opencensus/nodejs": "^0.0.15",
+    "@opencensus/nodejs": "^0.0.22",
     "@opencensus/propagation-tracecontext": "^0.0.15",
+    "@opentelemetry/core": "^0.10.1",
+    "@opentelemetry/api": "^0.10.1",
+    "@opentelemetry/node": "^0.10.2",
```

Comment:
This is a question to node.js experts: Why are these devDependencies? Should these rather be peer dependencies? How do we avoid importing both opencensus and opentelemetry if we're using only one?